packages/debug/src/node/debug-adapter.ts

```diff
@@ -64,6 +64,7 @@ export class LaunchBasedDebugAdapterFactory implements DebugAdapterFactory {
     }
 
     private childProcess(executable: DebugAdapterExecutable): RawProcess {
+        // tslint:disable-next-line:no-any
         const isForkOptions = (forkOptions: RawForkOptions | any): forkOptions is RawForkOptions =>
```

Comment:
I think it doesn't make sense to use `RawForkOptions | any` as the type... let's just use any?

Reply:
@thegecko what do you think? Should we simplify to simply using `any` in this case?

Reply:
I don't mind, whichever reads better :)

Reply:
I don't know, I just find it looks like `if (something || true)` :)

Reply:
Perhaps is should be `RawForkOptions | RawOptions`? Otherwise I'm happy making it just `any`

Reply:
@thegecko whatever you think is best :) would you like me to handle it?

Reply:
Sure, as you are in there already :)